### DISCUSSION TITLE
Loss of Talkgroup info when modifying user access

### DISCUSSION THREAD

**Discussion by AvalancheAddict** [chuot/rdio-scanner#128] [General]
I have, on multiple occasions lost my Talkgroup info while making modifications to the user access settings.

It appears that this is a database corruption issue, because I have noticed the following:

1. When this issue happens, the only Talkgroups that get wiped out are ones underneath Systems that I manipulated (by either turning on or off) TG's within those Systems.  Any Systems I did not mess with under the user access settings seems to be ok.  Basically if I turn on/off access to a specific TG underneath a system, there is a fair chance that entire System will lose all of it's TG's.
2. It seems that if I only turn on/off the entire System within the access section, it is ok.  The problems seem to only arise when I turn on/off individual TG's within the system, although it's not consistent enough for me to say that for sure.
3. The system itself doesn't get wiped out, it's just completely blank of TG's.  I do have automatically populate turned on, so basically when this happens, I just end up starting from scratch as it repopulates the TG's on it's own.
4. This issue is inconsistent.  It works normally most of the time, but sooner or later it crops up.  I have probably had it happen about 7 times now (sometimes I was better backup prepared than others :(

I am running version 6.1.10 on a Windows 2019 server, although I also had the issue with 6.1.8 (I never really messed with user access prior to 6.1.8).

Hopefully I explained this so that it is understandable.

Thank for looking into it.

**Comment by chuot** [comment]
Same issue as #127. The root cause has been identified yesterday and we are working on a fix. New version will be released in few hours if not tomorrow.

Thanks

**Reply by AvalancheAddict** [reply]
Thanks.  Glad you figured it out.  

I just installed 6.1.11 and made various changes in the access section which appeared to be the most likely triggering event for the issue and so far, so good.

Thanks again.
